CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/gather/GatherNodesContextMenu.java

```diff
@@ -64,15 +64,19 @@
     @Override
     public void selectItem(final String item, final Graph graph, final GraphElementType elementType, final int element, final Vector3f unprojected) {
         switch (elementType) {
-
             case GRAPH:
-                PluginExecution.withPlugin(new GatherNodesForGraphContextMenuPlugin(unprojected))
-                        .executeLater(graph);
+                PluginExecution.withPlugin(new GatherNodesForGraphContextMenuPlugin(unprojected)).executeLater(graph);
                 break;
-
             case VERTEX:
-                PluginExecution.withPlugin(new GatherNodesForVertexContextMenuPlugin(element))
-                        .executeLater(graph);
+                PluginExecution.withPlugin(new GatherNodesForVertexContextMenuPlugin(element)).executeLater(graph);
+                break;
+            case META:
+                break;
+            case LINK:
+                break;
+            case EDGE:
+                break;
+            case TRANSACTION:
```

Comment:
you can actually group all of these together if they do the same thing. i.e.
```
case META:
case LINK:
...
default:
   break;
```

Reply:
I added the breaks because it explicitly mentioned that it just breaks. More info in other review comments.